esx_service/vmdk_ops.py

```diff
@@ -708,7 +695,10 @@ def executeRequest(vm_uuid, vm_name, config_path, cmd, full_vol_name, opts):
     Returns None (if all OK) or error string
     """
     logging.debug("config_path=%s", config_path)
-    vm_datastore = get_datastore_name(config_path)
+    error_info, vm_datastore = vmdk_utils.get_datastore_name_from_config_path(config_path)
```

Comment:
I think Prashant is asking what's the value (not the format) of the config_path here - does it contain the old datastore name or the new one? I think it should contain the new (i.e. renamed) datastore name. If that's the case, our datastore cache should not contain this name. We should NOT force to refresh the cache unless the cache is missed.